pynicotine/gtkgui/ui/settings/uploads.ui

```diff
@@ -240,7 +240,7 @@
           <object class="GtkLabel">
             <property name="visible">True</property>
             <property name="xalign">0</property>
-            <property name="label" translatable="yes">Queue Limits</property>
```

Comment:
Would it be better to change `Upload Speed Limits` to `Speed Limits` instead, to avoid redundancy?

Reply:
The redundancy is needed to aid navigation, describing for instance "the Speed Limits in the Uploads section" is not as easy as referring to "the Upload Speed Limits".

Each section title should uniquely identify the relevant settings, without ambiguity across pages.

Reply:
On the other hand, the shorter sub-section headings are easier to read, see above screenshot.